guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,705 @@
 {
```

Comment:
<p>Can we introduce the following indentation</p><p><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.html#generate-trend" target="_blank">generate_trend()</a></code></p><ul><li>Adds options for&nbsp;<code>regression_type</code>&nbsp;parameter: </li><li class="ql-indent-1"><code>MANN-KENDALL</code></li><li class="ql-indent-1"><code>SEASONAL-KENDALL</code></li><li>Adds new parameter:</li><li class="ql-indent-1"><code>seasonal_period </code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Indentation fixed!